stable/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -2,10 +2,10 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ template "datadog.clusterAgent.fullname" . }}
+  name: {{ include "datadog.fullname" . }}-cluster-agent
   labels:
     app: "{{ template "datadog.fullname" . }}"
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    chart: {{ include "datadog.chart" . }}
```

Comment:
`{{ template "datadog.clusterAgent.fullname" . }}` needs to be updated here: https://github.com/helm/charts/pull/11903/commits/e8e3405a850ea2294f73d8557c4329409fed4280#diff-19938b6253581d6defa2db7d73496176R15
and here: 
https://github.com/helm/charts/pull/11903/commits/e8e3405a850ea2294f73d8557c4329409fed4280#diff-19938b6253581d6defa2db7d73496176R19
and here:
https://github.com/helm/charts/pull/11903/commits/e8e3405a850ea2294f73d8557c4329409fed4280#diff-19938b6253581d6defa2db7d73496176R118